packages/orderbook/src/orderbook.ts

```diff
@@ -246,7 +249,7 @@ export class Orderbook {
       );
     }
 
-    return this.seaport.fulfillOrder(orderResult, takerAddress, extraData);
+    return this.seaport.fulfillOrder(orderResult, takerAddress, extraData, amountToFill);
   }
 
   async fulfillBulkOrders(
```

Comment:
@Sam-Jeston Shouldn't we update the `fulfillBulkOrders` function also to support partial fulfilment ?

Reply:
indeed we should!